BDArmory/Modules/ModuleWeapon.cs

```diff
@@ -3233,7 +3210,7 @@ public void RunTrajectorySimulation()
                                 trajectoryPoints.Add(simCurrPos);
                             if (!aiControlled && !slaved)
                             {
-                                if (Physics.Raycast(simPrevPos, simCurrPos - simPrevPos, out hit, Vector3.Distance(simPrevPos, simCurrPos), layerMask1))
+                                if (simTime > 0.1f && Physics.Raycast(simPrevPos, simCurrPos - simPrevPos, out hit, Vector3.Distance(simPrevPos, simCurrPos), layerMask1))
```

Comment:
Why no raycasts within the first 0.1s? Surely this would mean that you could fire through walls if they're close enough.

Reply:
As part of the looking into aim reticle position stuff, I looked at how the rockets had been originally configured, back when it was still the RocketLauncher.cs class, and that's how it was done there; I figured Bahamuto had a reason. I note that the rocket class also doesn't start doing impact check raycasts until >0.1s flighttime, so presumably that has something to do with it. Hmm. That would indeed potentially allow shooting through walls. Rockets also don't start thrusting until 0.04s after launch. waiting to make sure everything was initialized properly? That might explain the 0.1s pause. Ah, legacy code.
Lets revert that back to what it was for now.